src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
I don't think assigning property value as `!( "msFullscreenElement" in document )` and then checking it in IE is real test for this bug.

As i said above, we could deal with `support` properties latter, right now, having test in integration folder while leaving comment that suggest to check specific test if someone would like to change relevant lines would be enough for me.